packages/utils/lib/jsonSchema/nangoModelToJsonSchema.ts

```diff
@@ -138,11 +145,13 @@ function nangoFieldToJsonSchema(field: NangoModelField): JSONSchema7 {
     };
 }
 
-const primitiveTypeMap: Record<string, JSONSchema7> = {
+const tsTypeMap: Record<string, JSONSchema7> = {
     number: { type: 'number' },
     bigint: { type: 'number' },
     boolean: { type: 'boolean' },
     string: { type: 'string' },
     date: { type: 'string', format: 'date-time' },
-    undefined: { type: 'null' } // There's no json schema mapping for this.
+    undefined: { type: 'null' }, // There's no json schema mapping for this.
+    any: {},
+    'Record<string, any>': { type: 'object' }
```

Comment:
would something like `Record<any, any>` or anything where the key is not a string ever representable?